libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -95,6 +107,31 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     AP_GROUPEND
 };
 
+AC_AttitudeControl::AC_AttitudeControl( AP_AHRS &ahrs,
+                    const AP_Vehicle::MultiCopter &aparm,
+                    AP_Motors& motors,
+                    float dt) :
+#if APM_BUILD_TYPE(APM_BUILD_ArduSub)
```

Comment:
AP_AttitudeControl_Sub, and then we rename AC_AttitudeControl to AP_AttitudeControl